packages/snaps-utils/src/manifest/validators/unused-exports.ts

```diff
@@ -0,0 +1,80 @@
+import type { InitialPermissions } from '@metamask/snaps-sdk';
+
+import type { ValidatorMeta } from '../validator-types';
+
+/**
+ * Check if all the required files are included.
```

Comment:
?

Reply:
Forgot to update when copying another validator. Fixed.